modern/src/LoginPage.js

```diff
@@ -2,60 +2,71 @@ import React, { useState } from 'react';
 import { useDispatch } from 'react-redux';
 import { useHistory } from 'react-router-dom';
 import { sessionActions } from './store';
-import Button from '@material-ui/core/Button';
-import FormControl from '@material-ui/core/FormControl';
-import Paper from '@material-ui/core/Paper';
-import { makeStyles } from '@material-ui/core';
-import TextField from '@material-ui/core/TextField';
+import { Grid, 
+         useMediaQuery, 
+         makeStyles, 
+         InputLabel, 
+         Select, 
+         MenuItem, 
+         FormControl, 
+         Button, 
+         TextField, 
+         Paper } from '@material-ui/core';
+import { useTheme } from '@material-ui/core/styles';
 import RegisterDialog from './RegisterDialog';
 import { useSelector } from 'react-redux';
 
 import t from './common/localization';
 
 const useStyles = makeStyles(theme => ({
   root: {
-    width: 'auto',
-    marginLeft: theme.spacing(3),
-    marginRight: theme.spacing(3),
-    [theme.breakpoints.up(400 + theme.spacing(3 * 2))]: {
-      width: 400,
-      marginLeft: 'auto',
-      marginRight: 'auto',
+    display: 'flex',
+    height: '100vh',
+  },
+  sidebar: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    background: theme.palette.common.purple,
+    width: '28%',
+    [theme.breakpoints.down('xs')]: {
+      width: '0px',
     },
   },
   paper: {
-    marginTop: theme.spacing(8),
-    display: 'flex',
+    display:'flex',
     flexDirection: 'column',
+    justifyContent: 'center',
     alignItems: 'center',
-    padding: theme.spacing(3),
-  },
-  logo: {
-    marginTop: theme.spacing(2)
+    width: '100%',
+    padding: theme.spacing(8, 4),
+    boxShadow: '-2px 0px 16px rgba(0, 0, 0, 0.25)'
```

Comment:
What this for?

Reply:
The main section of the login page has a shadow. It's not very obvious, but it helps with the visual appeal